src/sentry/utils/event_tracker.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from enum import IntEnum
+
+from sentry import options
+
+
+class EventType(IntEnum):
+    TRANSACTION = 1
+    ERROR = 2
+
+
+class TransactionStageStatus(IntEnum):
+    """
+    Do NOT change the int values of existing items. It will change meaning of existing tracking data
+    This class is DELETE or APPEND ONLY
+    """
+
+    # the transaction is stored to rc-transactions
+    REDIS_PUT = 1
```

Comment:
is there a reason for making these ints rather than strings or something else? are there size concerns?

personally i feel like this makes this code a lot less flexible and less easy to evolve

what if another stage between redis_put and save gets added later? will that then become 8? that's seems confusing.

Reply:
👍 for StrEnum for these, especially if they're being serialized in any way

Reply:
The reason originally was because it was planned to be stored in redis. smaller enums means higher sampling rate for the same memory usage. 
As we are not storing them in redis, there is no reason for int enum. Please turn them to strings.

Reply:
changed to strings